### PR TITLE
Fixed global filter delete function

### PR DESCRIPTION
**Pull request by jeudy100**
Fixed issue in #217

### CODE REVIEW

File: app/js/components/groupSelect/groupSelect.jade [hunk 1]
```diff
@@ -27,7 +27,7 @@
         a(href="#", ng-click='triggerChange(g)')
           trimmed-text(length="35", text="{{g.display_name}}")
           | &nbsp;({{g.systems.length}})
-          i.fa.fa-trash-o.pull-right.group-control.inline-left(ng-click='deleteGroup(g); $event.stopPropagation();',
```
Comment (jharting) [inline app/js/components/groupSelect/groupSelect.jade]: This was here intentionally. Without `stopPropagation` a groups is selected when the trash can icon is clicked, which is not expected. Instead the href should be removed from the hyperlink above, I think

**Comment by jharting** [comment]
Not directly related but since we're at this, here are some more CSS problems:

Trash can icon not alligned:
screenshot_20170906_150349

Some min width is needed, otherwise the placeholder is not readable.
screenshot_20170906_150620


Moreover, if we can get the input box to get focus once the group select is expanded that would be super handy. It would be enough to click "All systems" and start typing instead of having also to hit the search box.

**Comment by jeudy100** [comment]
@jharting i fixed the delete functionality and the styling issues, but i havent figured out how to make the search bar to be selected by default.
Would it be best to go ahead an merge this to fix the delete functionality and i can make another PR when i get the search bar to be selected by default?

**Comment by jharting** [comment]
@jeudy100 sure, works for me. I'll go ahead and review/merge this once the CI completes.

**Pull request merged**